src/termdict/termdict.rs

```diff
@@ -122,14 +130,8 @@ impl TermDictionary {
     }
 
     /// Creates an empty term dictionary which contains no terms.
-    pub fn empty(field_type: &FieldType) -> Self {
```

Comment:
There was in fact no dependence on knowing anything about `FieldType`. So, to show that the same data source can be used for all types of `TermDictionary`, I have removed this argument.

Reply:
yes. no clue what this was about